boxes/boxes/vite/src/config.ts

```diff
@@ -24,44 +20,15 @@ export class PrivateEnv {
   async init() {
     const nodeURL = process.env.AZTEC_NODE_URL ?? "http://localhost:8080";
 
+    const aztecNode = await createAztecNodeClient(nodeURL);
     const config = getPXEServiceConfig();
     config.dataDirectory = "pxe";
-    const aztecNode = await createAztecNodeClient(nodeURL);
-    const simulationProvider = new WASMSimulator();
-    const proofCreator = new BBWASMLazyPrivateKernelProver(
-      simulationProvider,
-      16,
-    );
     const l1Contracts = await aztecNode.getL1ContractAddresses();
     const configWithContracts = {
       ...config,
       l1Contracts,
     } as PXEServiceConfig;
-
-    const store = await createStore(
-      "pxe_data",
-      configWithContracts,
-      createLogger("pxe:data:idb"),
-    );
-
-    const keyStore = new KeyStore(store);
-
-    const db = await KVPxeDatabase.create(store);
-    const tips = new L2TipsStore(store, "pxe");
-
-    const protocolContractsProvider = new LazyProtocolContractsProvider();
-
-    this.pxe = new PXEService(
-      keyStore,
-      aztecNode,
-      db,
-      tips,
-      proofCreator,
-      simulationProvider,
-      protocolContractsProvider,
-      config,
-    );
-    await this.pxe.init();
+    this.pxe = await createPXEService(aztecNode, configWithContracts);
```

Comment:
oh wow this seems quite sane

Reply:
note the `pxe = pxe_service` strange assignment here though, as mentioned offline